EasyPost/Request.cs

```diff
@@ -103,7 +103,7 @@ public class Request {
                         result.AddRange(FlattenParameters(list[i], string.Concat(parent, "[", pair.Key, "][", i, "]")));
                     }
                 } else if (pair.Value is DateTime) {
-                    result.Add(new Tuple<string, string>(string.Concat(parent, "[", pair.Key, "]"), Convert.ToString(((DateTime)pair.Value).ToString("yyyy-MM-ddTHH:mm:sszzz"))));
+                    result.Add(new Tuple<string, string>(string.Concat(parent, "[", pair.Key, "]"), ((DateTime)pair.Value).ToLocalTime().ToString("yyyy-MM-ddTHH:mm:sszzz")));
```

Comment:
I just don't see how this would happen in normal shipment creation, that is why I am interested in a test case to reproduce the issue using the library.